README.md

```diff
@@ -66,7 +66,7 @@ instead using a custom `[tool.poetry]` table. This is partly because Poetry came
 
 ### [Hatch](https://hatch.pypa.io)
 
-Hatch can also manage environments (it allows multiple environments per project, but it does not allow to put them in the project directory), and it can manage packages (but without lockfile support). Hatch can also be used to package a project (with PEP 621-compliant pyproject.toml files) and upload it to PyPI.
+Hatch can also manage environments, allowing multiple environments per project. By default it has a central location for all environments but it can be configured to put a project's environment(s) in the project root directory. It can manage packages but without lockfile support. It can also be used to package a project (with PEP 621/631-compliant pyproject.toml files) and upload it to PyPI.
```

Comment:
```suggestion
Hatch can also manage environments, allowing multiple environments per project. By default it has a central location for all environments but it can be configured to put a project's environment(s) in the project root directory. It can manage packages but without lockfile support. It can also be used to package a project (with PEP 621-compliant pyproject.toml files) and upload it to PyPI.
```

From the PEP itself:

> This PEP has been accepted and was merged into [PEP 621](https://peps.python.org/pep-0621).

So I don't think 631 should be listed here :slightly_smiling_face: